src/Sentry/SentryClient.cs

```diff
@@ -137,7 +138,33 @@ public void CaptureTransaction(Transaction transaction, Hint? hint)
             return;
         }
 
-        var processedTransaction = BeforeSendTransaction(transaction, hint ?? new Hint());
+        scope ??= new Scope(_options);
+        hint ??= new Hint();
+        hint.AddAttachmentsFromScope(scope);
+
+        _options.LogInfo("Capturing transaction.");
+
+        scope.Evaluate();
+        scope.Apply(transaction);
+
+        _enricher.Apply(transaction);
+
+        var processedTransaction = transaction;
+        if (transaction.IsSampled != false)
```

Comment:
is this nullable? The fact it's not `!IsSampled` indicates so and it means if value is `null` we'll also go in the block.

I see it's the original code copied but wonder

Reply:
It's not nullable.